modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm_{context}']
+ifdef::virt-importing-vmware-vm[]
+= Importing a VMware virtual machine or template
+
+You can import a VMware virtual machine or template using the virtual machine wizard.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+= Importing a Red Hat Virtualization virtual machine
+
+You can import a Red Hat Virtualization (RHV) virtual machine using the virtual machine wizard.
+endif::[]
+
+.Prerequisites
+
+ifdef::virt-importing-vmware-vm[]
+* You must create a VDDK image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
+* The VMware virtual machine must be powered off.
+endif::[]
+
```

Comment:
For RHV to CNV need this requirement for RHV-4.3 or RHV-4.4 onld clusters as mentioned here:
Bug 1843822 - [v2v][Doc] RHV to CNV VM import: Change VM bios in RHV from i440fx to q35

Reply:
Also in the troubleshooting, we can add that if VM import fails on:

"The virtual machine could not be imported.
MappingRulesVerificationFailed: VM uses unsupported bios type: i440fx_sea_bios"
Then it's related to this requirement.